pacman-nodejs-app/tools/migrate/pacman.sh

```diff
@@ -45,24 +62,37 @@ function set_new_mongo_primary {
 
 # TODO: make DNS management generic enough for all applications
 function update_pacman_dns {
-    gcloud dns record-sets transaction start -z=${ZONE_NAME}
-    gcloud dns record-sets transaction remove -z=${ZONE_NAME} \
-        --name="pacman.${DNS_NAME}" --type=A --ttl=1 "${PACMAN_SRC_PUBLIC_IP}"
-    gcloud dns record-sets transaction add -z=${ZONE_NAME} \
-        --name="pacman.${DNS_NAME}" --type=A --ttl=1 "${PACMAN_DST_PUBLIC_IP}"
-    gcloud dns record-sets transaction execute -z=${ZONE_NAME}
+    	gcloud dns record-sets transaction start -z=${ZONE_NAME}
+	echo ${PACMAN_SRC_PUBLIC_ADDRESS}
+	if  valid_ip ${PACMAN_SRC_PUBLIC_ADDRESSS} ; then 
+		echo "did ip"
```

Comment:
this is where the problem is. Still working it out but will delete this when I have